change/@fluentui-react-charting-f8073f84-d26a-4b28-992e-379202a63f5d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33476&commitId=26f00aeaa43eb76c1947cb5598b13042defedf07&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>